client/src/components/postcall/PostCallOneQuestionPoll.tsx

```diff
@@ -20,10 +20,11 @@ export const PostCallOneQuestionPoll: React.FunctionComponent<PostCallOneQuestio
   props: PostCallOneQuestionPollProps
 ) => {
   const [pollResponse, setPollResponse] = useState<boolean | string | number>();
+  const [isSubmittingResponse, setIsSubmittingResponse] = useState<boolean>(false);
 
   const submitSurveyResponse = async (): Promise<void> => {
     const { acsUserId, callId, meetingLink } = props;
-
+    setIsSubmittingResponse(true);
     await submitSurveyResponseUtil(acsUserId, pollResponse, meetingLink, callId);
```

Comment:
We might want to add a try catch here to setIsSubmittingResponse(false) if the submitSurveyResponseUtil() call throws an exception. Otherwise if exception is thrown, UI will be stuck spinning and user cannot try re-submitting. For example:

```
setIsSubmittingResponse(true);
try
{
    await submitSurveyResponseUtil(acsUserId, pollResponse, meetingLink, callId);
    window.location.replace('/book');
}
catch (e)
{
    setIsSubmittingResponse(false);
}
```

Reply:
I see your point, but I do have a try...catch within the **submitSurveyResponseUtil** function. Also since we have window redirect I feel that this code may not really execute.

Reply:
I like it here because it allows UI to not have to depend on the internal implementation of submitSurveyResponseUtil and UI is more resilient to future changes. But up to you if you want to change it or leave it this way.

Reply:
hmm Ok, have made this change! thanks!